### DISCUSSION TITLE
Can I use openapi-fetch to fetch from an unspecified endpoint?

### DISCUSSION THREAD

**Discussion by johassl14** [openapi-ts/openapi-typescript#2050] [Q&A]
Is it possible to use OpenAPI fetch without defining the path, similar to using the standard fetch function?

The reason I don't just use fetch is that I have some middleware defined that I would still like to utilize.

Thanks for the help!

**Comment by drwpow** [comment]
Not really; if you are fetching from arbitrary endpoints, you are by definition not using an OpenAPI schema so you don’t need to use openapi-fetch. But as a one-off workaround, you could always use middleware to change the URL of specific requests to whatever you want. But this behavior is intentionally hard to do because the library is designed to not let people do this easily (which lead to bugs)
